src/components/navigation/AlertDropdown.tsx

```diff
@@ -42,13 +42,15 @@ export function AlertDropdown(props: Props) {
   })
 
   return (
-    <AirshipDropdown autoHideMs={persistent ? 0 : undefined} bridge={bridge} backgroundColor={color} onPress={handleOnPress}>
+    <AirshipDropdown autoHideMs={persistent ? 0 : undefined} bridge={bridge} backgroundColor={color}>
```

Comment:
I don't understand this change. Is the commit message wrong?

Edit: Ok, maybe the idea is that we want the X to close the modal, but tapping the text should open the link?